data/users.json

```diff
@@ -1,6 +1,7 @@
 [
   "David_Leal_7",
   "telmo",
+  "_Gurveen_"
```

Comment:
```suggestion
  "_Gurveen_",
```

Reply:
Sorry man I deleted the fork, so I'll make a new PR